config/gasSchedule.go

```diff
@@ -174,7 +176,7 @@ func FillGasMap_ElrondAPICosts(value uint64) map[string]uint64 {
 	gasMap["DelegateExecution"] = value
 	gasMap["ExecuteReadOnly"] = value
 	gasMap["AsyncCallStep"] = value
-	gasMap["AsyncCallbackGasLock"] = value
+	gasMap["AsyncCallbackGasLock"] = AsyncCallbackGasLock
```

Comment:
Replace the global constant with a parameter to `FillGasMap_ElrondAPICosts()`, and to `FillGasMap()` and `MakeGasMap()` as well. `MakeGasMap()` is also used in some integration tests in `elrond-go`, where it will need to be updated, but it's better anyway, because there was no way to customize it previously.

An alternative would be to remove the global, add _no_ new parameter, but instead to change the value of `AsyncCallbackGasLock` in the map returned by `MakeGasMap()` in the tests that need it. If it's sufficiently easy, I recommend this alternative, as opposed to the global constant and to the new parameter. Then there's no update needed in `elrond-go`.